pages/common/fakedata.md

```diff
@@ -0,0 +1,24 @@
+# fakedata
+
+> CLI utility for fake data generation.
```

Comment:
That it's a CLI utility is made obvious by the fact the tldr-page (tries to) only document CLI tools.  How about this instead?
```suggestion
> Generate fake data using a large variety of generators.
```